packages/hash/frontend/src/blocks/page/BlockView.tsx

```diff
@@ -268,6 +269,10 @@ export class BlockView implements NodeView<Schema> {
                   ctx?.setShowDataMappingUi(!ctx.showDataMappingUi)
                 }
               />
+              <CommentButton
+                blockId={blockEntityId}
+                className={styles.Block__Comments_Button!}
+              />
```

Comment:
Why do we pass `className` instead of defining these styles inside `CommentButton`, since it renders a `Box` component inside. If it's because we want to keep the block UI styles inside `packages/hash/frontend/src/blocks/page/style.module.css`, then let's move the absolute styles inside `packages/hash/frontend/src/blocks/page/BlockHighlight.tsx` to `style.module.css` as well.

_I'm aware of this comment not being directly related with this PR. But if you agree, at least we can create a asana ticket to not forget moving the related styles (if that's desider). Let me know what do you think_

Reply:
Addressed this here: https://github.com/hashintel/hash/pull/1060#discussion_r971109760